packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -230,18 +239,52 @@ export class BucketDeployment extends CoreConstruct {
       }
     }
 
+    if (props.useEfs && !props.vpc) {
+      throw new Error('Vpc must be specified if useEfs is set');
+    }
+
+    const accessPointPath = '/lambda';
+    let accessPoint;
+    if (props.useEfs && props.vpc) {
+      const accessMode = '0777';
+      const fileSystem = this.getOrCreateEfsFileSystem(scope, {
+        vpc: props.vpc,
+        removalPolicy: cdk.RemovalPolicy.DESTROY,
+      });
+      accessPoint = fileSystem.addAccessPoint('AccessPoint', {
+        path: accessPointPath,
```

Comment:
So the EFS is a singleton, but all instances would end up using the same hard-coded path in there... That sounds possibly risky... Maybe the `accessPointPath` should contain the non-singleton construct `addr`?

Reply:
The generated path for different lambdas in one VPC will be of format `/lambda/1234-4567-1234(random UUID)`. A random UUID will be appended to constant path.